contrib/kava-3/migration.md

```diff
@@ -77,27 +114,45 @@ kvd start --halt-time 1591794000
   # commit: 869189054d68d6ec3e6446156ea0a91eb45af09c
   # build_tags: netgo,ledger
   # go: go version go1.13.7 linux/amd64
+```
+
+3. Migrate the kava-2 keys from previous key store to new key store
 
+Thill will scan for any keys in `.kvcli` and produce new files ending in `kavaxxx.address` and `key_name.info` for the new keystore to access.
```

Comment:
```suggestion
This will scan for any keys in `.kvcli` and produce new files ending in `kavaxxx.address` and `key_name.info` for the new keystore to access.
```